app/lib/widgets/quantity_selector/quantity_selector.dart

```diff
@@ -34,22 +34,26 @@ class FlexQuantitySelector extends StatelessWidget {
     final canDecrement = quantity - increment >= minQuantity;
     final canIncrement = quantity + increment <= maxQuantity;
 
+    final theme = Theme.of(context);
+    // Fallback theme styling if IconButtonTheme.style is not set by library user
+    final themeStyle = Theme.of(context).iconButtonTheme.style ?? ButtonStyle();
+
     return Row(
       mainAxisSize: MainAxisSize.min,
       children: <Widget>[
         IconButton(
           padding: EdgeInsets.zero,
-          style: Theme.of(context).iconButtonTheme.style!.copyWith(
-                padding: const WidgetStatePropertyAll(EdgeInsets.zero),
```

Comment:
Only thing Changed here was:
``Theme.of(context).iconButtonTheme.style!`` => ``themeStyle``

Rest of code change indication is just format shift